cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -555,8 +556,21 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
         Map<String, Object> optimizationProposal = new HashMap<>();
         optimizationProposal.put(CruiseControlRebalanceKeys.SUMMARY.getKey(),
                 proposalJson.getJsonObject(CruiseControlRebalanceKeys.SUMMARY.getKey()).getMap());
-        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
-
+//        optimizationProposal.put(BROKER_LOAD_KEY, beforeAndAfterBrokerLoad);
+
+        String mapAsString = beforeAndAfterBrokerLoad.keySet().stream()
+                .map(key -> key + "\n" + converttoformat(beforeAndAfterBrokerLoad.get(key)) + "\n")
+                .collect(Collectors.joining(", ", "{", "}"));
+        ConfigMap rebalanceMap = new ConfigMapBuilder()
+                .withApiVersion("v1")
+                .withNewMetadata()
+                .withName("kafka-rebalance-before-after-load")
+                .withLabels(Collections.singletonMap("app", "strimzi"))
+                .endMetadata()
+                .withData(Collections.singletonMap(BROKER_LOAD_KEY, mapAsString))
+                .build();
+        KubernetesClient kubernetesClient = new DefaultKubernetesClient();
+        kubernetesClient.configMaps().inNamespace("myproject").createOrReplace(rebalanceMap);
```

Comment:
The name here you should get from the `KafkaRebalance` instance for this reconciliation